core/src/main/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulation.kt

```diff
@@ -489,7 +489,7 @@ fun getStopPositions(schedule: List<SimulationScheduleItem>): List<Double> {
  * Converts a DistanceRangeMap<T> into a legacy RangeMap<Double, T>. Distances are converted to
  * floats (m).
  */
-private fun <T> DistanceRangeMap<T>.toRangeMap(): RangeMap<Double, T> {
+fun <T> DistanceRangeMap<T>.toRangeMap(): RangeMap<Double, T> {
```

Comment:
nit: if it's not local to this file we should probably move it somewhere appropriate for `DistanceRangeMap` tooling. Like directly in the class.